### PR TITLE
MNT: Add space to pylab examples with figsize and/or tight_layout

### PR DESCRIPTION
**Pull request by sargas**
As nice as http://matplotlib.org/examples/pylab_examples/contour_image.html makes matplotlib look, I added tight_layout to a handful (and other minor adjustments) to them less crowded (particularly the tick layouts)

### CODE REVIEW

**Comment by tacaswell** [comment]
Can you use the `tight_layout=True` kwarg in the figure constructor instead?  I think that will let it work with the osx backend (see https://github.com/matplotlib/matplotlib/issues/1852).

attn @mdehoon Am I correct that `fig = plt.figure(tight_layout=True)` works where as `fig.tight_layout()` does not?

**Comment by mdehoon** [comment]
@tacaswell  On master, all flavors of tight_layout should now work with the MacOSX backend. Issue #1852 cannot be closed yet though, as it still relies on get_renderer, which is not implemented on most backends. (the tight_layout code works because it gets the cached renderer and doesn't need to call get_renderer).

**Comment by tacaswell** [comment]
But that relies on the figure having been drawn at least once?

File: examples/pylab_examples/contour_image.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 norm = cm.colors.Normalize(vmax=abs(Z).max(), vmin=-abs(Z).max())
 cmap = cm.PRGn
 
-plt.figure()
+plt.figure(figsize=(10,12))
```
Comment (jenshnielsen) [inline examples/pylab_examples/contour_image.py]: The Travis failure is due to PEP8, There is a missing space after the , as in `figsize=(10, 12)`

**Comment by dstansby** [comment]
Instead of trying to rebase, I've manually made the changes in #8985

**Comment by tacaswell** [comment]
Taken care of by  #8985

@sargas Sorry this sat for so long and thank you for your work!

**Pull request closed without merging**